### PR TITLE
Workspace Homepage

### PR DESCRIPTION
**Pull request by matt-paul**
[Ticket]()

[Zeplin](https://app.zeplin.io/project/5f1852163d4a8e79eb8592de/dashboard?seid=5f565ec91be47e78cc9ab802)

[Figma](https://www.figma.com/file/tYFsqhG61CleWS1tUapAar/Source-of-Truth-UI?node-id=20%3A2&viewport=494%2C721%2C0.33870282769203186)

As a member
I want to see a workspace homepage with an activity feed
So that I know what is most recent content

## Outcome/acceptance criteria
Acceptance criteria:
- [x] Website can show a homepage for every workspace
- [x] Workspace homepage requires users to be signed in
- [x] Workspace homepage shows workspace name

NOTE: E2E Testing following in next PR as needs investigation to get around azure auth.

## Testing information

NOTE: there is a bug in the workspace service.  You may need to restart it using the following command to avoid a socket error.
`kubectl rollout restart deploy -n workspace-service`

Port forward to the workspace-service in your cluster with the following command:
`sudo kubefwd svc -n workspace-service `

- Test plan?
  - Select a workspace id from your database
  - Go to localhost:3000/workspaces/<ID_FROM_ABOVE>
  - Vefify that the information on the page is correct.

## Test:

- [ ] Tested locally

- Platforms/Browsers:

  - [ ] Google Chrome (latest version)

  - [ ] Internet Explorer 11

  - [ ] Edge

- Accessibility:

  - [ ] Screen Reader

  - [ ] Keyboard Navigation

  - [ ] Magnification

  - [ ] Contrast

  - [ ] Text size 200%

  - [ ] Touch target areas (Mobile)

  - [ ] Landscape (Mobile)

### CODE REVIEW

File: frontend/pages/workspaces/workspaces.graphql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+query GetWorkspaceByID($id: ID!) {
+  workspace(id: $id) {
+    id
```
Comment (frigus02) [inline frontend/pages/workspaces/workspaces.graphql:3]: I don't think we use the `id` at the moment. Should we remove it from the query response?

File: frontend/pages/workspaces/[id].tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import { Header } from "../../components/Header";
+import { MainHeading } from "../../components/MainHeading";
+import { PageLayout } from "../../components/PageLayout";
+import { requireAuthentication } from "../../lib/auth";
+import { getSdk } from "../../lib/generated/graphql";
+
+interface Workspace {
+  id: string;
+  title: string;
+  longDescription: string;
```
Comment (frigus02) [inline frontend/pages/workspaces/[id].tsx:17]: This doesn't match the columns returned from the GraphQL query. I wonder how to make it match. I think you can use `GetServerSideProps<Props>` as the type for your server side props function. Then it should complain.

**Review by frigus02 — COMMENTED** [review]
2 initial comments. I can review the styles later. Looks good, though

File: frontend/pages/workspaces/workspaces.graphql (hunk 1 above)
Comment (matt-paul) [reply frontend/pages/workspaces/workspaces.graphql:3]: makes sense

**Review by frigus02 — APPROVED** [review]

**Pull request merged**